### PR TITLE
Continuous integration addition: kubejs recipe parsing

### PR DESCRIPTION
**Pull request by Quezler**
Leaving this here for @NielsPilgaard to review when he has time.

Basically this pull will cause ci to spit out nameless (`:kjs_`) kubejs recipes as well as recipe id's added by our kubejs scripts & being overwritten by us again somehow. (aka: ci will point us into their general direction)

This might be the first but certainly won't be the last of the kubejs related ci thingies, but it will be able to be built upon and expanded.

Screen Shot 2022-05-15 at 18 49 56

(ps, squash this commit or you'll get 37 worthless commit messages)

### CODE REVIEW

File: .github/actions/kubejs-recipes.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    groups[group][id] = recipe;
+  });
+
+  // write each group to their own file
+  for (const [group, recipes] of Object.entries(groups)) {
+    fsp.writeFile(`kubejs/exported/recipes/${group}.json`, JSON.stringify(recipes, null, '\t'));
+  }
+
+  console.log(`::notice::${Object.entries(groups['nameless']).length} KubeJS recipes without a randomized name, and ${Object.entries(groups['duplicate']).length} duplicates.`)
```
Comment (NielsPilgaard) [inline .github/actions/kubejs-recipes.js:60]: I think this should be with a randomized name :D

**Review by NielsPilgaard — APPROVED** [review]

File: .github/actions/kubejs-recipes.js (hunk 1 above)
Comment (Quezler) [reply .github/actions/kubejs-recipes.js:60]: Whoops, you're right 😅

Comment (Quezler) [reply .github/actions/kubejs-recipes.js]: fixed 👍

**Review by NielsPilgaard — APPROVED** [review]

**Comment by Quezler** [comment]
rip, no squash merge 👀

**Comment by NielsPilgaard** [comment]
We don't squash around here :D

**Comment by Quezler** [comment]
Guess i'll have to make my ci testing commit messages a little less nonsensical since all of them show up here 🙊

**Comment by NielsPilgaard** [comment]
I don't know, I like them as-is :D

**Pull request merged**